### DISCUSSION TITLE
License for site and calculator usage

### DISCUSSION THREAD

**Discussion by cutaway** [cutaway-security/IACS_STAR_Methodology#15] [Ideas]
I just realized the site does not have a license. I'm not sure the best one to pick for its use, especially since this will be used internationally. I'm looking for recommendations. Here are my thoughts:

- Need a general license file in the root directory.
- Custom calculator script needs a license statement.
- The calculator page needs a license statement / reference at the bottom. Should be included on visible page.
- Methodology write-up should include a reference to the license.

I am hoping that @c-bless, @davisjessa28, @workentin, and @Lijantropique will provide input. I'll point others here to comment. Feel free to do the same.

**Comment by tliston** [comment]
The OWASP stuff is licensed under the MIT license (https://opensource.org/license/mit). I'm thinking that since this is based on that and is intended to be used in much the same way, it would make sense to use that.

**Reply by cutaway** [reply]
This makes sense. Since the calculator was initially based on it, I'll need to re-read that license.

**Reply by c-bless** [reply]
I would agree with MIT license.
